scripts/license/add.py

```diff
@@ -16,22 +16,29 @@
 Adds a license header to files.
 """
 
+import os
 from pathlib import Path
 import subprocess
 
 from license_header import license_header, has_license_header
 
-PROJECT_ROOT = Path(subprocess.check_output(
-    "git rev-parse --show-toplevel".split()).strip().decode("utf-8"))
+PROJECT_ROOT = subprocess.check_output(
+    "git rev-parse --show-toplevel".split()).strip().decode("utf-8")
+target_dirs = [os.path.join(PROJECT_ROOT, d) for d in os.listdir(PROJECT_ROOT)
+               if d != "vendor" and os.path.isdir(os.path.join(PROJECT_ROOT, d))]
 
 
 def main(verbose=False):
-    add(PROJECT_ROOT.glob("*[!vendor]/**/*_k8s.go"),
-        license_header("//", True), verbose)
-    add([p for p in PROJECT_ROOT.glob("*[!vendor]/**/*.go")
-         if p.name[-7:] != "_k8s.go"], license_header("//"), verbose)
-    add(PROJECT_ROOT.glob("*[!vendor]/**/*.py"), license_header("#"), verbose)
-    add(PROJECT_ROOT.glob("*[!vendor]/**/*.sh"), license_header("#"), verbose)
+    add(Path(PROJECT_ROOT).glob("*.go"), license_header("//"), verbose)
+    add(Path(PROJECT_ROOT).glob("*.py"), license_header("#"), verbose)
+    add(Path(PROJECT_ROOT).glob("*.sh"), license_header("#"), verbose)
+
+    for d in target_dirs:
+        add(Path(d).glob("**/*_k8s.go"), license_header("//", modification=True), verbose)
+        add([p for p in Path(d).glob("**/*.go")
+             if p.name[-7:] != "_k8s.go"], license_header("//"), verbose)
```

Comment:
For avoiding a magic number ..
```suggestion
             if not p.name.endswith("_k8s.go")], license_header("//"), verbose)
```
(I wrote the original code :wink: )